### PR TITLE
feat(frontend): Implement Open Graph Protocol tm-304

### PR DESCRIPTION
**Pull request by GEOFARL**
image

### CODE REVIEW

File: apps/frontend/index.html [hunk 1]
```diff
@@ -29,6 +29,24 @@
...
+
+		<meta property="og:type" content="website" />
+		<meta
+			property="og:url"
+			content="http://development.eu-central-1.elasticbeanstalk.com/"
+		/>
+		<meta property="og:title" content="Trackmates" />
+		<meta
+			property="og:description"
+			content="Trackmates - monitor your progress and the progress of your friends!"
```
Comment (what1s1ove) [inline apps/frontend/index.html:41]: as QA about the better description

File: apps/frontend/index.html [hunk 2]
```diff
@@ -29,6 +29,24 @@
 		<meta name="msapplication-TileColor" content="#da532c" />
 		<meta name="theme-color" content="#ffffff" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+		<meta property="og:type" content="website" />
+		<meta
+			property="og:url"
+			content="http://development.eu-central-1.elasticbeanstalk.com/"
```
Comment (what1s1ove) [inline apps/frontend/index.html:36]: we can have more than 1 env

File: apps/frontend/index.html [hunk 3]
```diff
@@ -29,6 +29,24 @@
...
+		/>
+		<meta property="og:title" content="Trackmates" />
```
Comment (what1s1ove) [inline apps/frontend/index.html:38]: also change the general title and description

File: apps/frontend/public/assets/screen-desk-1200-630.png
Comment (what1s1ove) [inline apps/frontend/public/assets/screen-desk-1200-630.png:1]: the name should be `socials.png`

Comment (what1s1ove) [inline apps/frontend/public/assets/screen-desk-1200-630.png:1]: on the screen we should see our logo and that it (we discard it during our today's call)

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by Vfivew** [comment]
Why do you use large size 1200*630 png? Perhaps you need an SVG or a slightly cropped copy?

File: apps/frontend/index.html [hunk 4]
```diff
@@ -29,7 +29,26 @@
...
+			content="Trackmates - monitor your progress and the progress of your friends!"
+		/>
+		<meta property="og:image" content="/assets/socials.png" />
+
+		<meta property="og:site_name" content="Trackmates" />
+		<meta property="og:locale" content="en_US" />
+		<meta property="og:image:width" content="1200" />
+		<meta property="og:image:height" content="630" />
+
+		<meta
```
Comment (what1s1ove) [inline apps/frontend/index.html:47]: should be before the og

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**